downstream/modules/platform/proc-installing-the-aap-setup-bundle.adoc

```diff
@@ -64,16 +61,16 @@ automationhub_pg_username='automationhub'
 automationhub_pg_password='dbpassword123'
 automationhub_pg_sslmode='prefer'
 ----
-NOTE: The inventory should be kept intact after installation since it is used for backup, restore, and upgrade functions.  Consider keeping a backup copy in a secure location, given that the inventory file contains passwords.
+NOTE: The inventory should be kept intact after installation because it is used for backup, restore, and upgrade functions. Consider keeping a backup copy in a secure location, given that the inventory file contains passwords.
 +
-. Run the AAP setup bundle executable as the root user
+. Run the {PlatformNameShort} setup bundle executable as the root user:
 +
 ----
 $ sudo -i
 # cd /path/to/ansible-automation-platform-setup-bundle-2.4-1
 # ./setup.sh
 ----
 +
-. Once installation is complete, navigate to the Fully Qualified Domain Name (FQDN) for the Automation controller node that was specified in the installation inventory file.
+. When installation is complete, navigate to the Fully Qualified Domain Name (FQDN) for the {ControllerName} node that was specified in the installation inventory file.
```

Comment:
Check with @dcdacosta about the FQDN. For some reason, I thought this had changed.

Reply:
I'm not aware of a change with regard to FQDN? 🤔

Reply:
Okay. I think I was peer reviewing someone else's PR and they replaced this value. Ignore my comment above.